wemake_python_styleguide/visitors/ast/operators.py

```diff
@@ -283,21 +290,16 @@ def visit_BinOp(self, node: ast.BinOp) -> None:
             BitwiseAndBooleanMixupViolation
 
         """
-        self._check_BitOp(node)
+        self._check_logical_bitwise_operator(node)
         self.generic_visit(node)
 
-    def _check_BitOp(self, node: ast.BinOp) -> None:
+    def _check_logical_bitwise_operator(self, node: ast.BinOp) -> None:
         if not isinstance(node.op, (ast.BitOr, ast.BitAnd)):
             return
 
-        if (self._check_sides(node.left) or self._check_sides(node.right)):
+        if self._check_sides(node.left) or self._check_sides(node.right):
             self.add_violation(BitwiseAndBooleanMixupViolation(node))
 
     # checks either side of the Bitwise operation invalid usage
     def _check_sides(self, node) -> bool:
```

Comment:
Let's better name it `_is_bool_like`. What do you think?

Reply:
Sure, that's a would be more descriptive of what the function actually does.